### DISCUSSION TITLE
Channels DVR should be moved to "Media" section

### DISCUSSION THREAD

**Discussion by johnsturgeon** [community-scripts/ProxmoxVE#82] [General]
I noticed the Channels DVR lxc is in the PVR DVR section and I think it belongs with apps like Plex

**Comment by remz1337** [comment]
or in both (making a case to convert categories to tags...)

**Comment by johnsturgeon** [comment]
Why was this closed?  It's still in "Cameras and NVR"  That's just wrong.

**Reply by MickLesk** [reply]
feel free to do an PR for this, 5 Month old Discussion is marked as outdated

**Reply by johnsturgeon** [reply]
OK, cool

**Reply by johnsturgeon** [reply]
https://github.com/community-scripts/ProxmoxVE/pull/3729
